area-data/concentrations/africa-and-the-americas.yaml

```diff
@@ -0,0 +1,99 @@
+name: Africa and the Americas
+type: concentration
+revision: 2014-15
+result: Interdisciplinary Seminar & four of (Independent, Electives)
+
+Interdisciplinary Seminar:
+    Africa and the Americas: AFAM 231
+
+Independent:
+    Academic Internship II: AFAM 294
+    Academic Internship III: AFAM 394
+    Independent Study: AFAM 298
+    Directed Undergraduate Research: AFAM 396
```

Comment:
Ah ha.

You can't have both an implicit `result` and an explicit `message` key in the same requirement.

Just change it to 

```yaml
Directed Undergraduate Research: 
    result: AFAM 396
    message: ...
```